lib/app/modules/addOrUpdateAlarm/views/ascending_volume.dart

```diff
@@ -0,0 +1,147 @@
+// ignore_for_file: lines_longer_than_80_chars
+
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:ultimate_alarm_clock/app/modules/addOrUpdateAlarm/controllers/add_or_update_alarm_controller.dart';
+import 'package:ultimate_alarm_clock/app/modules/settings/controllers/theme_controller.dart';
+import 'package:ultimate_alarm_clock/app/utils/constants.dart';
+import 'package:ultimate_alarm_clock/app/utils/utils.dart';
+
+class AscendingVolumeTile extends StatelessWidget {
+  const AscendingVolumeTile({
+    Key? key,
+    required this.controller,
+    required this.themeController,
+  }) : super(key: key);
+
+  final AddOrUpdateAlarmController controller;
+  final ThemeController themeController;
+
+  @override
+  Widget build(BuildContext context) {
+    int gradient;
+    return InkWell(
+      onTap: () {
+        Utils.hapticFeedback();
+        gradient = controller.gradient.value;
+        Get.defaultDialog(
+          onWillPop: () async {
+            Get.back();
+            // Resetting the value to its initial state
+            controller.gradient.value = gradient;
+            controller.selectedGradientDouble.value = gradient.toDouble();
+
+            return true;
+          },
+          titlePadding: const EdgeInsets.symmetric(vertical: 20, horizontal: 5),
+          backgroundColor: themeController.isLightMode.value
+              ? kLightSecondaryBackgroundColor
+              : ksecondaryBackgroundColor,
+          title: 'Volume will reach maximum in '.tr,
```

Comment:
The corresponding translation hasn't been added in utils/languages

Reply:
I'll add them.